.rubocop.yml

```diff
@@ -171,6 +175,27 @@ Layout/CaseIndentation:
 Style/CharacterLiteral:
   Enabled: false
 
+Style/AccessModifierDeclarations:
+  Enabled: false
+
+Style/TrailingCommaInArrayLiteral:
+  Enabled: false
+
+Style/TrailingCommaInHashLiteral:
+  Enabled: false
+
+Style/UnneededCondition:
+  Enabled: false
```

Comment:
should this be true? seems like a good thing to catch..

Reply:
with this enabled:
```lib/vanagon/common/user.rb:7:24: C: Style/UnneededCondition: Use double pipes || instead.
        @group = group ? group : @name
                       ^^^^^^^^^
lib/vanagon/platform.rb:250:7: C: Style/UnneededCondition: This condition is not needed.
      if instance_variable_get("@#{key}") ...
      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^```

Reply:
oh weird